### PR TITLE
only GmfLut & NcLutModel

### PR DESCRIPTION
**Pull request by vincelhx**
my change concern the xsarsea PR also that delete sarwing luts

### CODE REVIEW

**Comment by vincelhx** [comment]
I think we need to merge #81 from @Skealz at the same time

File: grdwindinversion/.travis.yml
Comment (agrouaze) [inline grdwindinversion/.travis.yml:1]: `travis` is supposed to help for CI tests, I think we don't use it in `grdwindinversion`, 
Also, I suspect that `agrouaze/grdwindinversion` is the initial repo URL... 
I suggest to remove it, unless you plan to use it in the future.

**Review by agrouaze — CHANGES_REQUESTED** [review]
`travis` to remove.

**Comment by Skealz** [comment]
> I think we need to merge #81 from @Skealz at the same time

Don't wait for the merge of my PR

File: grdwindinversion/.travis.yml
Comment (vincelhx) [reply grdwindinversion/.travis.yml:1]: done

**Review by agrouaze — APPROVED** [review]

**Pull request merged**